pkg/model/crd.go

```diff
@@ -304,6 +304,25 @@ func (r *CRD) IsPrimaryARNField(fieldName string) bool {
 		strings.EqualFold(fieldName, r.Names.Original+"arn")
 }
 
+// GetCustomSetOutputOperation returns custom set output operation as string for
+// given operation on custom resource, if specified in generator config
+func (r *CRD) GetCustomSetOutputOperation(
```

Comment:
You could just call this method `SetOutputCustomMethodName()`

Additionally, consider making the attribute a `*string` instead of `string` and returning `nil` here instead of `""` to indicate there is no custom method name...

Reply:
Updated the method name.